swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/tagging/EntityTag.kt

```diff
@@ -55,3 +56,9 @@ open class EntityTagRequest(
   open val application: String,
   open val description: String
 ) : TagRequest
+
+open class SwabbieEntityTag {
```

Comment:
you can probably just do 
`const val SWABBIE_ENTITY_TAG_NAME = "spinnaker_ui_alert:swabbie_deletion_candidate"`
in this file. What is the use of the class with a companion object?

Reply:
i added it as a class with an companion object to access it in swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/tagging/OrcaTaggingService.kt

Reply:
nvm , fixed it